numba/np/random/generator_methods.py

```diff
@@ -819,3 +820,75 @@ def impl(inst, n, p , size=None):
                 out[i] = random_negative_binomial(inst.bit_generator, n, p)
             return out
         return impl
+
+
+@overload_method(types.NumPyRandomGeneratorType, 'noncentral_chisquare')
```

Comment:
Need to add all these methods to the documentation.

Reply:
Resolved in [9bcf3e2](https://github.com/numba/numba/pull/8520/commits/9bcf3e2edbdb4f1fdb5417ccada72d589b6da88e)